docs/csharp/programming-guide/generics/constraints-on-type-parameters.md

```diff
@@ -18,22 +18,31 @@ Constraints inform the compiler about the capabilities a type argument must have
 
 |Constraint|Description|
 |----------------|-----------------|
-|`where T : struct`|The type argument must be a non-nullable [value type](../../language-reference/builtin-types/value-types.md). For information about nullable value types, see [Nullable value types](../../language-reference/builtin-types/nullable-value-types.md). Because all value types have an accessible parameterless constructor, the `struct` constraint implies the `new()` constraint and can't be combined with the `new()` constraint. You can't combine the `struct` constraint with the `unmanaged` constraint.|
+|`where T : struct`|The type argument must be a non-nullable [value type](../../language-reference/builtin-types/value-types.md), including `record struct` types. For information about nullable value types, see [Nullable value types](../../language-reference/builtin-types/nullable-value-types.md). Because all value types have an accessible parameterless constructor, either declared or implicit, the `struct` constraint implies the `new()` constraint and can't be combined with the `new()` constraint. You can't combine the `struct` constraint with the `unmanaged` constraint.|
```

Comment:
```suggestion
|`where T : struct`|The type argument must be a non-nullable [value type](../../language-reference/builtin-types/value-types.md), which includes `record struct` types. For information about nullable value types, see [Nullable value types](../../language-reference/builtin-types/nullable-value-types.md). Because all value types have an accessible parameterless constructor, either declared or implicit, the `struct` constraint implies the `new()` constraint and can't be combined with the `new()` constraint. You can't combine the `struct` constraint with the `unmanaged` constraint.|
```